config.py

```diff
@@ -3,25 +3,25 @@
 
 class Config:
     SQLALCHEMY_TRACK_MODIFICATIONS = True
-    SECRET_KEY = os.getenv("SECRET_KEY")
+    SECRET_KEY = os.getenv("SECRET_KEY", default="Do I really need this?")
```

Comment:
I don't think you need a default
it should be provided by environment variable.
you can set a dummy value in `DevelopmentConfig`